presentation-compiler/src/main/dotty/tools/pc/completions/Completions.scala

```diff
@@ -107,7 +106,7 @@ class Completions(
   end includeSymbol
 
   def completions(): (List[CompletionValue], SymbolSearch.Result) =
-    val (advanced, exclusive) = advancedCompletions(path, pos, completionPos)
+    val (advanced, exclusive) = advancedCompletions(path, completionPos.originalCursorPosition, completionPos)
```

Comment:
```suggestion
    val (advanced, exclusive) = advancedCompletions(path, completionPos)
```
And to avoid too many changes in `advancedCompletions ` just do:
```
val pos = completionPos.originalCursorPosition
```